draft-ietf-quic-recovery.md

```diff
@@ -577,15 +577,22 @@ Pseudocode for OnPacketSent follows:
 
 ### On Ack Receipt
 
-When an ack is received, it may acknowledge 0 or more packets.
+When an acknowledgment is received, it may acknowledge 0 or more packets.
+
+The first acknowledgment for the largest acknowledged packet is used to update
```

Comment:
nit: "first acknowledgement received for", just to clarify that it's the first one we've seen, not the first one sent.